### DISCUSSION TITLE
[Feature] Adhoc Load Original Image/Video

### DISCUSSION THREAD

**Discussion by antonio-ivanovski** [immich-app/immich#11513] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Recently I have started recording videos in 4k 60fps leading, this caused my server upload speed not to be able to keep up with the bitrate. So I have enabled video transcoding (recently I had it disabled since my videos were 1080p 30fps/60fps and the upload speed could keep up with the bitrate).

With this sometimes I see the need to stream the video in the full glory. By using the "Download" feature is not great option since the video could be multiple GB. This is because A. there might be not space on my device, and B. I don't want to wait for the entire video to download just for me to skip to 5s I am interested in.

Because of this I think there should be option in the photo/video viewer that will allow us to see/watch photos/videos in the original state. This should be on best-effort basis, immich not caring if the app support display/playback of the original file format whatever it might be.

<img width="253" alt="image" src="https://github.com/user-attachments/assets/7ebf5177-09fa-4db7-9667-8f986e9bc561">

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by antonio-ivanovski** [comment]
After further searching, found this is duplicate of https://github.com/immich-app/immich/discussions/6306

**Reply by Notesuser** [reply]
I don't think this feature has been covered before. The thread mentioned is already closed and referred to photo.

I would really welcome this feature for videos too. I like to collect photos and videos in an Immich album for presentations. The photos are then shown very well on a large display when I load the originals, but the videos are only mediocre. It would be great if you could also see the videos in the original.
